flake.nix

```diff
@@ -1,6 +1,6 @@
 {
   inputs = {
-    nixpkgs.url = "github:NixOS/nixpkgs?ref=nixos-21.05";
+    nixpkgs.url = "github:NixOS/nixpkgs/nixpkgs-unstable";
```

Comment:
Pinning to unstable for user facing flakes is not a good idea

Reply:
OK, I've switched back to 22.05.